src/GafferUI/AnimationGadget.cpp

```diff
@@ -75,17 +75,21 @@ namespace AnimationLayer
 	constexpr Gadget::Layer Overlay = Gadget::Layer::Front;
 };
 
-// \todo: Use actual fps setting. For now, we always use 24 fps
 template<typename T>
-T frameToTime( T frame )
+T frameToTime( float fps, T frame )
```

Comment:
This seems a bit error prone. Which way round do the arguments go? My guess would have been `frame` first. I'd be tempted to turn these into member functions, so they can use `m_context` directly and the caller doesn't need to pass anything extra at all. It's all internal though, so if you want to leave that as future refactoring, that's fine...

Reply:
Yeah, I was wondering about that, too. 

I think making it a member would require `computeGrid()` to be a member, too. Maybe I was more hesitant than I should have been to just move both into the class because I remember doing some restructuring at some point so that they could be anonymous. Maybe it's not a lot of clutter in the class, after all...

Reply:
Maybe eventually the grid drawing code including `computeGrid()` should live closer to the `ViewportGadget`? I'm just thinking that other editors/viewers might need something like that, as well. A `UVViewer` comes to mind and maybe there's more...